core/tests/protractor_desktop/additionalEditorFeaturesModals.js

```diff
@@ -380,6 +381,57 @@ describe('Full exploration editor', function() {
       await users.logout();
     });
 
+  it(
+    'should be able to save changes when offline',
```

Comment:
I thought we didn't let users save changes when offline (the save changes button gets disabled). Instead, we let users make changes while offline and then save them when they're back online, right?

Reply:
yes right, Changed the description.